tests/test_web_app_handler.py

```diff
@@ -50,7 +51,8 @@ def test_not_completed_workflow():
     assert http_code == 200
 
 
-def test_missing_workflow_job():
+def test_missing_workflow_job(monkeypatch):
+    monkeypatch.delenv("GH_APP_ID", raising=False)
```

Comment:
When I do local development, I sometimes use `pytest` directly, thus, it will read my `.env` variables and think that I'm executing within the context of a Github app.